src/fns/unconstrained_ops.nr

```diff
@@ -171,34 +171,38 @@ pub(crate) unconstrained fn __udiv_mod<let N: u32>(
     let mut divisor_u60: U60Repr<N, 2> = U60Repr::from(divisor);
     let b = divisor_u60;
 
-    let mut bit_difference = remainder_u60.get_msb() - divisor_u60.get_msb();
```

Comment:
I see how this line can cause issues when the denominator has more bits in its u60 representation. Thanks for catching this!